glean-core/ios/Glean/Glean.swift

```diff
@@ -29,6 +29,8 @@ public class Glean {
         static let logTag = "glean/Glean"
     }
 
+    private var pingTypeQueue: [Ping] = []
```

Comment:
This is a big nit and definitely could just be biased towards a particular style, but I typically use the initializer syntax for arrays like this:
```
private var pingTypeQueue = [Ping]()
```

Your way is perfectly valid though. So again, just a style nit.

Reply:
Hm, Let's use it.